src/app/styles/js/shared.js

```diff
@@ -389,6 +348,21 @@ export const HeaderTitle = styled.h3`
   `}
 `;
 
+export const StyledSubHeader = styled.h2`
+  font: ${title1};
+  font-weight: 600;
+  color: ${black54};
+  text-align: center;
+  margin-top: ${units(2)};
+`;
+
+export const StyledCard = styled(Card)`
+  padding: ${units(11)} ${units(15)} ${units(3)} !important;
+  ${mediaQuery.handheld`
+    padding: ${units(8)} ${units(4)} ${units(3)} !important;
+  `}
+`;
+
```

Comment:
Fine for now but would be best to use a dedicated component under `components/layout`. Also naming should ideally state a clear purpose for the component. `StyledCard` is a bit vague. I'd suggest creating a `CheckBrandedCard` component.
Please add a FIXME comment.

Reply:
Added FIXME comment